latex/minted/minted.dtx

```diff
@@ -1805,6 +1805,10 @@
   \endgroup
 \fi
 %    \end{macrocode}
+% To unify the structure of this macro to be able to compare it using |\ifx| it has to be converted to a string.
+%    \begin{macrocode}
+\xdef\minted@timestamp{\expandafter\detokenize\expandafter{\minted@timestamp}}
```

Comment:
The `\expandafter` before `\detokenize` is redundant. There are 3 such cases added by this commit.

The form
```tex
\xdef\minted@timestamp{\detokenize\expandafter{\minted@timestamp}}
```
is already used in 15cd9144ac507235c1ef4aad7ea86985a9e8a7f8.

Reply:
you are right - thats fine then.